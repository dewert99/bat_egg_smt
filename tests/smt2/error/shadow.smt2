(declare-fun f () Bool)
(declare-fun f (Bool) Bool)