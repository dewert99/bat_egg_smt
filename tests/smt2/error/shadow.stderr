the identifier `f` shadows a global constant at 2:14
(declare-fun f (Bool) Bool)
             ^

